README.md

```diff
@@ -11,8 +15,101 @@ scram b -j 8
 The c++ interface require you to scram b after checkout. If you do not place the code in the above hierarchy within CMSSW
 the python paths are not guaranteed to work.
 
+# Supported Trigger Scale Factors and Measurements
+The tau trigger SFs are provided separately for the different offline DeepTau ID working points. They are provided for the three different recommended trigger combinations in the root files in the data directory of this repository.
+The recommended triggers are:
+
+* Mu+Tau Cross Trigger:
+    * 2016:
+        * HLT_IsoMu19_eta2p1_LooseIsoPFTau20_SingleL1
+    * 2017:
+        * HLT_IsoMu20_eta2p1_LooseChargedIsoPFTau27_eta2p1_CrossL1
+    * 2018:
+        * HLT_IsoMu20_eta2p1_LooseChargedIsoPFTau27_eta2p1_CrossL1,     for Run < 317509
+        * HLT_IsoMu20_eta2p1_LooseChargedIsoPFTau**HPS**27_eta2p1_CrossL1,  for Run >= 317509
+
+* Elec+Tau Cross Trigger:
+    * 2016:
+        * HLT_Ele24_eta2p1_WPLoose_Gsf_LooseIsoPFTau20_SingleL1,    for Run < 276215
+        * HLT_Ele24_eta2p1_WPLoose_Gsf_LooseIsoPFTau20,             for 276214 < Run < 278270
+        * HLT_Ele24_eta2p1_WPLoose_Gsf_LooseIsoPFTau30,             for Run > 278269
+    * 2017:
+        * HLT_Ele24_eta2p1_WPTight_Gsf_LooseChargedIsoPFTau30_eta2p1_CrossL1
+    * 2018:
+        * HLT_Ele24_eta2p1_WPTight_Gsf_LooseChargedIsoPFTau30_eta2p1_CrossL1, for Run < 317509
+        * HLT_Ele24_eta2p1_WPTight_Gsf_LooseChargedIsoPFTau**HPS**30_eta2p1_CrossL1,  for Run >= 317509
+
+* di-Tau Triggers: OR of all fully enabled triggers in 2017 data for Run < 317509 before HPS tau reconstruction is deployed, and use the single ditau trigger for Run >= 317509 after HPS is deployed
```

Comment:
I find this sentence a bit confusing. It might not be clear what is meant by fully enabled triggers but I dont have a good suggestion for a better wording. Maybe it could be good to split into two sentences describing before and after Run < 317509.
```suggestion
* di-Tau Triggers: 
    * For 2016, 2017 and 2018 data for Run < 317509: OR combination of all fully enable triggers listed below
    * For 2018 data for Run >= 317509: Use the single ditau trigger after HPS is deployed
```

Reply:
I tries to state it more clearly by making the description a little more verbose.